sql/new-install/MinimalData.sql

```diff
@@ -5,7 +5,7 @@ INSERT INTO users (email, first_name, last_name, institution, created, password,
 	('admin@uiowa.edu', 'Admin', 'User', 'The University of Iowa', SYSDATE(), 'b109f3bbbc244eb82441917ed06d618b9008dd09b3befd1b5e07394c706a8bb980b1d7785e5976ec049b46df5f1326af5a2ea6d103fd07c95385ffab0cacbc86', 107374182400),
 	('public', 'Public', 'User', 'None', SYSDATE(), 'd32997e9747b65a3ecf65b82533a4c843c4e16dd30cf371e8c81ab60a341de00051da422d41ff29c55695f233a1e06fac8b79aeb0a4d91ae5d3d18c8e09b8c73', 52428800);
 
-INSERT INTO user_roles VALUES
+--INSERT INTO user_roles VALUES
```

Comment:
This is an incorrect change and needs to be reverted before we can accept this PR.  Commenting out the start of a sql command does not comment out the rest of the command.  Furthermore, StarExec needs to have a public user in order to operate correctly.  

We are guessing you commented this out because of the uiowa domain, but there is no real working email address associated with these users (admin@uiowa.edu is not a real email address that we have access to).